components/SubmissionForm.vue

```diff
@@ -1,162 +1,172 @@
-        <textarea
-            v-model="submissionStore.otherNotes"
-            data-testid="submit-input-notes"
-            class="mb-5 landscape:mb-20 px-3 py-3.5 w-96 h-28 bg-secondary-bg rounded-lg border border-primary-text-muted"
-            maxlength="300"
-        />
-        <button
-            data-testid="submit-submitbutton"
-            type="submit"
-            class="px-20 py-3 mb-40 landscape:mb-0 rounded-full bg-primary w-96 text-center
-             text-primary-inverted font-medium font-sans"
-            @click="validateFields"
-        >
-            {{ $t('submitPage.submitButton') }}
-        </button>
-    </form>
+                {{ $t('submitPage.invalidOption') }}
+            </p>
+            <select
+                v-model="submissionStore.selectLanguage2"
+                data-testid="submit-select-language2"
+                class="mb-5 px-3 py-3.5 w-96 h-12 bg-primary-text-inverted rounded-lg border border-primary-text-muted
+                        text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+                :placeholder="$t('submitPage.selectLanguage2')"
+                @change="initialValidationCheck(submissionStore.selectLanguage1, 'secondaryLanguage')"
+            >
+                <option
+                    v-for="(locale, index) in localeStore.localeDisplayOptions"
+                    :key="index"
+                    selected
+                    :value="locale.code"
+                >
+                    {{ locale.displayText }}
+                </option>
+            </select>
+            <span
+                class="mb-2 text-primary-text text-sm font-normal font-sans"
+            >{{ $t('submitPage.otherNotes') }}({{ $t('submitPage.optional') }})</span>
+            <textarea
+                v-model="submissionStore.otherNotes"
+                data-testid="submit-input-notes"
+                class="mb-5 landscape:mb-20 px-3 py-3.5 w-96 h-28 bg-secondary-bg rounded-lg border border-primary-text-muted"
+                maxlength="300"
+            />
+            <button
+                data-testid="submit-submitbutton"
+                type="submit"
+                class="px-20 py-3 mb-40 landscape:mb-0 rounded-full bg-primary w-96 text-center
+                 text-primary-inverted font-medium font-sans"
+                @click="validateFields"
+            >
+                {{ $t('submitPage.submitButton') }}
+            </button>
+        </form>
+    </div>
 </template>
 
 <script lang="ts" setup>
 import { ref, watch, nextTick } from 'vue'
 import * as validations from '../utils/formValidations'
 import { useSubmissionStore } from '~/stores/submissionStore'
 import { useLocaleStore } from '~/stores/localeStore'
+import { useLoadingStore } from '#imports'
 
 const submissionStore = useSubmissionStore()
 const localeStore = useLocaleStore()
+const loading = useLoadingStore()
+loading.checkStoresInstalled([submissionStore, localeStore])
```

Comment:
@ermish it is empty refs within the submission store. They are always empty unless you want us to add sessionStorage in case they refresh the page with things filled in.

There is no data to fetch in these stores.

It is similar to how you are using suspense in the auth. On refresh of the `submit` path why do we care if the rest of the data is fetched since they are refreshing directly on that page? The refs in the store are not async